www/search.spt

```diff
@@ -14,16 +14,17 @@ if query:
     q = strip_accents(query)
 
     if action in (None, 'search_usernames'):
-        results['usernames'] = website.db.all("""
-            SELECT username, avatar_url, similarity(username, %(q)s) AS rank
-              FROM participants
-             WHERE username %% %(q)s
-               AND claimed_time IS NOT NULL
-               AND is_searchable
-               AND NOT is_closed
-          ORDER BY rank DESC, username
-             LIMIT 10
-        """, locals())
+         results['usernames'] = website.db.all("""
+             SELECT username, avatar_url, similarity(username, %(q)s) AS rank
+               FROM participants
+              WHERE username %% %(q)s
+                AND claimed_time IS NOT NULL
+                """
+                +("" if user.ADMIN else " AND is_searchable")
```

Comment:
Hmm ... could we find a way to do this without string interpolation? (Tbh this is the achilles heel of our db wiring; it's bad practice to construct SQL from strings because of the danger of SQL injection, but it's hard to avoid without a full-blown SQL model in Python, aka SQLAlchemy.)